iscsi/initiator.go

```diff
@@ -23,6 +24,10 @@ var (
 
 const (
 	iscsiBinary = "iscsiadm"
+	manualMode = "manual"
+	autoMode = "auto"
+	getScanModeTimeout = time.Second
```

Comment:
One second is too short. The load might be high in some environments. We need to prepare for that. I would say at least 10 seconds.